doc/architecture/environmental_effect_model.adoc

```diff
@@ -0,0 +1,9 @@
+= Environmental effect model
+
+Environmental effect models consume sensor-view data and produce sensor-view data.
+Environmental effect models may, for example, include the following:
+
+* Shadows and occlusions
+* Weather effects
+* Physics of a sensor
+* Pre-processing of raw sensor data
```

Comment:
Plural: I used plural throughout. Sounded better than using indefinite articles.

Hyphens: In English compounds are usually not coupled via hyphens. However, if compounds are used as attributes, it is good practice to use hyphens to avoid ambiguity and increase legibility (Example: notice the subtle difference between "I wear a single-use glove." vs. "I wear a single use glove." ).

So in our case: We write "sensor view" without a hyphen when we talk about the thing (as in "the sensor view"). But, we write "sensor-view messages" because in this case "sensor-view" is used like a single adjective.